### DISCUSSION TITLE
How does strongswan scep client determine if AES128-CBC is supported

### DISCUSSION THREAD

**Discussion by mikaellove** [strongswan/strongswan#1702] [General]
Hello!

We have a scep server implementation that supports AES according to https://datatracker.ietf.org/doc/html/rfc8894#section-3.5.2.
Listing the ca capabilities with GetCaCaps the AES keyword is returned as specified in the RFC8894. 

Our issue, when a request with the strongswan scep client using the cipher aes is performed the following message is returned:
"AES_CBC encryption algorithm not supported by CA"

Might there be something in our implementation that we are missing since your client does not accept the AES keyword we are sending with GetCaCaps.
How does strongswan scep client determine whether or not AES128-CBC encryption algorithm is supported?

**Comment by tobiasbrunner** [comment]
Please see #1607.
